feature-libs/asm/assets/translations/en/asm.ts

```diff
@@ -32,6 +32,16 @@ export const asm = {
       submit: 'Start Session',
       noMatch: 'No customer found.',
     },
+    bindCart: {
+      cartNumber: 'Cart Number',
+      bindCartToCustomer: 'Assign Cart to Customer',
+      success: 'Cart has been successfully assigned',
+      aria: {
```

Comment:
I don't think we make a distinction for aria labels anywhere.  For consistency, perhaps remove the aria level.

Reply:
For the Iceman working on this ticket, this is a change I made.